olm-catalog/serverless-operator/project.yaml

```diff
@@ -38,3 +38,9 @@ dependencies:
   eventing_kafka_broker_artifacts_branch: release-v1.2
   cli: 1.2.0
   operator: 1.2.2
+  # Previous versions required for downgrade testing
+  previous:
```

Comment:
This is fine, just curious is there a way to get previous in some other way?

Reply:
I'm not aware of a good way to get that info. It might also be beneficial to have this info explicit here. Whenever someone updates the "current" version they should also update the "previous" version in the same pull request.
But, if you're aware of a better way let me know.

Reply:
I was thinking of the olm bundle image or something similar but dont think it is better, even if it was possible. Better be explicit.